.github/workflows/ci.yml

```diff
@@ -35,3 +39,21 @@ jobs:
       - uses: dtolnay/rust-toolchain@1.70.0
       - uses: Swatinem/rust-cache@v2
       - run: cargo check --workspace
+
+  nix-flake:
+    name: Test build Nix Flake
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: |
+          git fetch --prune --tags ||:
+      - uses: cachix/install-nix-action@v21
+        with:
+          extra_nix_config: |
+            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

Comment:
this is the default

```suggestion
```

Reply:
Thanks, I was unaware that had changed.

Reply:
for reference this was added in v19

https://github.com/cachix/install-nix-action/commit/a4b8aaf5a24260b643f571235829e53e6b3cc920